e2e/etcd_spawn_cov.go

```diff
@@ -55,9 +46,48 @@ func spawnCmd(args []string) (*expect.ExpectProcess, error) {
 		ep.StopSignal = syscall.SIGTERM
 		return ep, nil
 	}
+
+	if args[0] == ctlBinPath {
+		covArgs, err := getCovArgs()
+		if err != nil {
+			return nil, err
+		}
+		apiVersion := os.Getenv("ETCDCTL_API")
+		// set ctl command args as a env var
+		// where the ctl will read and run it.
+		// this is a hack to get around test binary flag conflicts
+		ctl_cov_env := []string{
+			"ETCD_CTL_ARGS" + "=" + strings.Join(args, " "),
```

Comment:
Is the `ETCD_CTL` because it conflicts with the environment variable to flag code? If there's no conflict it should be `ETCDCTL_ARGS`. If there is a conflict, `ETCDCTLCOV_ARGS`?